test/cql/suite.yaml

```diff
@@ -1,3 +1,6 @@
 type: Approval
+extra_scylla_config_options:
+    authenticator: AllowAllAuthenticator
```

Comment:
As per commit message, the `auth` keyspace starts with `RF=1` and if we are stopping nodes this can cause failures. For now topology tests don't need `auth` so I just disabled it. If in the future topology tests need `auth`, we can switch back to normal auth and we will have to do `ALTER KEYSPACE ... RF = 3` and run repair and wait for it to finish. But this will bring slowdown so we might want to split it.

> but here you also change it for cql suite.

Ouch, I could swear I removed it. Will do next.